telemetry/appblock.go

```diff
@@ -0,0 +1,105 @@
+package telemetry
+
+import (
+	"runtime"
+	"strconv"
+	"time"
+
+	abci "github.com/cometbft/cometbft/abci/types"
+	"github.com/cosmos/cosmos-sdk/baseapp"
+	"github.com/cosmos/cosmos-sdk/telemetry"
+	sdk "github.com/cosmos/cosmos-sdk/types"
+	"github.com/cosmos/cosmos-sdk/types/mempool"
+	"github.com/hashicorp/go-metrics"
+)
+
+type BlockTelemetry struct {
+	startTime            time.Time
+	initialBlockMemAlloc uint64
+	blockTxsSize         int
+}
+
+// InitBlockMetrics initializes the block metrics for the application
```

Comment:
`by initializing a BlockTelemetry struct` 

We really don't need this level of detail in comments.
1. It doesn't help
2. It makes things worse by increasing cognitive overhead
3. It gets stale.